ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/contentassist/TypePackageCompletionProcessor.java

```diff
@@ -214,14 +214,14 @@ public void acceptType(int modifiers, char[] packageName, char[] simpleTypeName,
 					// Accept search results from the JDT SearchEngine
 					String cName = new String(simpleTypeName);
 					String pName = new String(packageName);
-					String label = cName;
+					StringBuilder label = new StringBuilder().append(cName);
 					String replaceString = cName;
```

Comment:
Another curious case. Why was `lable` StringBuilderified but replaceString wasn't?

Reply:
I suggest to open a JDT UI bug for this cleanup.